src/UILayer/CQRSStoreGroup.ts

```diff
@@ -14,6 +14,36 @@ import { ChangedPayload } from "../payload/ChangedPayload";
 import { Dispatcher } from "../Dispatcher";
 const CHANGE_STORE_GROUP = "CHANGE_STORE_GROUP";
 
+
+/**
+ * This function is type converter for TypeScript.
+ * It has actual implementation, but it is only use for typing.
+ *
+ * - Status: unstable
+ * 
+ * This is useful for TypeScript, but it is unstable API.
+ * You can use it own risk.
+ * 
+ * ## Example
+ *
+ * ```ts
+ * const mapping = {
+ *   a: new AStore(),
+ *   b: new BStore()
+ * };
+ * const StoreState = MapStoreToState(mapping);
+ * // Get Mapped to State for Store
+ * type States = typeof StoreState
+ * // {
+ * //   a: AState,
+ * //   b: BState
+ * // };
+ * ```
+ */
+export function MapStoreToState<T>(_mapping: MapStore<T>): MapState<T> {
+    return {} as MapState<T>
+}
```

Comment:
> It has actual implementation, but it is only used for typing.

As a result, `MapStoreToState` does exist in JavaScript code.
I want that it should only be in TypeScript code.
But, I have no idea :(

![almin 2017-04-20 00-42-38](https://cloud.githubusercontent.com/assets/19714/25188837/5b3d66e0-2562-11e7-9256-63723d45534b.png)

@kimamula Do you think about it?